### DISCUSSION TITLE
Confused in power modes

### DISCUSSION THREAD

**Discussion by chockokruasan** [seerge/g-helper#2426] [Q&A]
Hello, I have read eveything on G-Helper page, but havn't found answers on my question.
The default power modes "silent", "balanced" and "turbo" in g-helper are the same as AC each power modes? They a factory written in bios correct? But what happening when I make some changes in factory power modes, does it become like "custom" or G-helper overwrite my changes over factory power mode? 
For example. I like to use factory "Balanced" because I like not high CPU clock speeds and lower GPU power, but I want to increase fan speeds a little. If I apply custom fun curves for factory "balanced" mode, it is become "cutom mode" and now all presets of "balanced" reseted to max (CPU clock speeds, power limits/package etc.)?
Or another one example. When I use "turbo" mode for heavy apps it has max fan speeds 5200rpm and my CPU temp going up to 100 degrees. I want to apply custom fan to 6200rpm max to stay at 92 degress in same work. I can also apply lower power limit, but it decrease performance too much.
Thanks!
Sorry for bad english :) Can I ask on russian here?

**Comment by seerge** [comment]
@chockokruasan 

> But what happening when I make some changes in factory power modes, does it become like "custom" or G-helper overwrite my changes over factory power mode?

Modes are baked in BIOS, they can't be overwritten. If you apply custom power limits or fan curves you get same result as you would get by setting ``Manual`` mode in Armoury.

**Reply by chockokruasan** [reply]
But what happens with power limits? In AC manual mode I need to set 3 lines of power limits and custom fans. When I choose for example "turbo" mode in G-Helper and apply custom fan curves, what happens with CPU power limits if I don't touch this section? Thanks

**Reply by seerge** [reply]
Each mode comes with some default fan curves and power limits (defined by manufacturer) and baked in BIOS. 

If you don't override default power limits - they stay same as they were in that mode.
